app/views/movies/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= @movie.title %></br>
+<%= @movie.creation_year %></br>
+<%= @movie.genre %></br>
+<%= @movie.list_actors %></br>
+Average Actor Age: <%= @movie.ave_actor_age %> years</br>
+
+Add an Actor
+<%= form_with method: :post, local: true do |form| %>
+  <%= form.label :actor_id %>
+  <%= form.number_field :actor_id %>
+  <%= form.hidden_field(:value => params[:actor_id]) %>
```

Comment:
Look closely - you are asking the user for the `actor_id` in line 10, but you are providing this hidden_field with the value of a `params[:actor_id]` which is not correct. You should instead be providing it with `@movie.id`. As such, this is likely only working because your test data is limited in its creation of movies-to-actors.

Reply:
You also forgot the `url:` in your `form_with` field; luckily it will default to the same location, but you could have avoided this issue if you included `url: "/movies/#{@movie.id}"`.